worldedit_commands/safe.lua

```diff
@@ -1,3 +1,9 @@
+local area_protection = {}
+
+if minetest.get_modpath("areas") then
+    area_protection.areas = areas
+end

```

Comment:
Yes, I'd prefer to have the interface implemented in a way so it can easily support any protection mod server maintainers may choose to use together with WorldEdit. I agree that the WorldEdit commands should better deal with an abstraction rather than directly calling the mod API, though.